docs/installation/presto-admin-upgrade.rst

```diff
@@ -4,9 +4,9 @@ Upgrading Presto-Admin
 
 Upgrading to a newer version of ``presto-admin`` requires deleting the old
```

Comment:
The process for upgrading from older versions of presto-admin will be different because the config directories have changed.  This can be done in a following pr though